.github/workflows/docstring_tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Docstrings
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # Allow this workflow to be called from other repositories.
+  workflow_call:
+
+# This workflow need Poetry, so no flag is included.
+
+jobs:
+  test-docstrings:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
```

Comment:
Do we want to explicitly specify the python version here (as is done in other workflows)? Does it install the latest version by default? If yes, that might cause issues somewhere, like we're currently having with 3.13.

Reply:
I thought about it, but I thought this docstring test is mostly a convenience thing, so we should make it low maintenance. Not having to update this, is less maintenance, but it can indeed lead to problems. I'll volunteer to add a specific Python version if this indeed becomes a problem.